grub-theme-endeavouros-classic/PKGBUILD

```diff
@@ -0,0 +1,22 @@
+# Base Maintainer: Filipe Laíns (FFY00) <lains@archlinux.org>
+# Original Sources: https://github.com/Se7endAY/grub2-theme-vimix; https://github.com/endeavouros-team/grub2-theme-endeavouros
+# Maintainer: LeSnake04 (dev.lesnake@gmail.com)
+pkgname=grub2-theme-endeavouros-classic
+pkgver=20210226
+pkgrel=1
+pkgdesc='EndeavourOS grub2 theme'
+arch=(any)
+url='https://github.com/endeavouros-team/'
+license=('GPL3')
+depends=('grub')
+optdepends=('grub-customizer: GUI tool to configure grub')
+replaces=('vimix-grub' 'grub-theme-vimix' 'grub-themes-vimix')
+conflicts=('grub2-theme-endeavouros')
+makedepends=('git')
+source=('git+https://github.com/lesnake04/grub2-theme-endeavouros-classic')
```

Comment:
wrong url

Reply:
Works for me.
And I have copied the formatting from the pkgbuild for  grub-theme-endeavouros

Reply:
the url needs to be source=('git+https://github.com/endeavouros-team/grub2-theme-endeavouros-classic')

Reply:
pkgdesc='EndeavourOS grub2 theme'
--> pkgdesc='EndeavourOS grub2 theme classic'

Reply:
i put it here for now:

https://github.com/EndeavourOS-Community-Editions/grub2-theme-endeavouros-classic